content/docs/0.6.0/usecases/_index.md

```diff
@@ -11,6 +11,7 @@ The above list of tutorials helps you to explore the functionalities of Keptn ba
 * As a **Developer** you might be most interested in:
   * [Onboarding a Service](./onboard-carts-service/)
   * [Deployments with Quality Gates](./deployments-with-quality-gates/)
+  * [Keptn Quality Gates only](./quality-gates/)
```

Comment:
I believe the title should be more like "Keptn Quality Gates Standalone" or maybe even just "Quality Gates standalone"

Reply:
I heard that the term "standalone" should not be used. Let's discuss this in a broader round.